blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/UpdateStamp.java

```diff
@@ -49,16 +61,27 @@ public boolean equals(Object o) {
     }
     
     public String toString() { return timestamp + "," + state.code; }
-    
+
+    /**
+     * The current state of a slot.
+     *
+     * In the database, there are only two states we care about: {@code Active}
+     * and {@code Rolled}. {@code Running} is a ephemeral state during runtime.
+     * It degrades to {@code Active} during a save restore (indicating it is
+     * not finished and should, therefore, not be considered rolled.
+     */
     public enum State {
-        // in the database, there are only two states we care about: Active and Rolled.  Running is a ephemeral state
-        // during runtime.  It degrades to Active during a save restore (indicating it is not finished and should,
-        // therefore, not be considered rolled.
-        Active("A"), Running("A"), Rolled("X");
+
+        Active("A"),
+        Running("A"),
```

Comment:
Yes, I left that value exactly as it originally appeared. I didn't write the enum or the comment, I just re-formated them. My impression is that the state gets serialized to the value when persisted to the database, and then when it gets pulled out of the database it gets deserialized to `Active`. I think that's what it means when it says "degrades to Active during a save restore".